bodhi/server/config.py

```diff
@@ -227,10 +227,10 @@ class BodhiConfig(dict):
             'value': None,
             'validator': _validate_none_or(str)},
         'admin_groups': {
-            'value': ['proventesters', 'security_respons', 'bodhiadmin', 'sysadmin-main'],
+            'value': ['proventesters', 'security_response', 'bodhiadmin', 'sysadmin-main'],
```

Comment:
It's possible this is tied to a database and thus this change could be problematic. Dropping this commit is easily done.

Reply:
practically speaking this can't be changed w/ FAS as it exists today, but this fact can be documented: #3133